app/models/open_food_network/calculator/weight.rb

```diff
@@ -4,7 +4,7 @@ class Calculator::Weight < Spree::Calculator
     attr_accessible :preferred_per_kg
 
     def self.description
-      "Weight (per kg)"
+      I18n.t('.spree.weight')
```

Comment:
Should this be just `I18n.t('spree.weight')`? But why is it in the spree namespace?